data_sharing.md

```diff
@@ -0,0 +1,55 @@
+# Data Sharing
+
+If a collaborator wants to upload data to a GCS bucket, you can follow this
+process to set up a service account that can be used to upload the data on their
+behalf:
+
+## Step 1: Create service account
+
+- Log into your GCP account and select the project that contains the bucket in
+  which the data will be uploaded.
+- `IAM & Admin` `->` `Service Accounts` `->` `Create Service Account`.
+- Fill in the service account name - the ID should be filled in automatically.
+- (Optional) Add a description of the service account.
+- `Create`.
+
+<img src="https://i.postimg.cc/dV7XQ6sX/create-service-account.png" alt="create service account" height="230" />
```

Comment:
Can we upload this to GitHub (in the `figures` folder), so we don't rely on an image posting service?